### DISCUSSION TITLE
https://mobx-react.js.org is offline!?

### DISCUSSION THREAD

**Discussion by jeggers-88** [mobxjs/mobx#3602] [General] [answered]
Hello,

I am new to MobX, it looks really attractive! I have one problem: In the Readme.md you often mention the site https://mobx-react.js.org, for example in this section: 
https://github.com/mobxjs/mobx/tree/main/packages/mobx-react#provider-and-inject 

Does anybody what is going on with https://mobx-react.js.org ? I would be very interested in the documentation there...

Best regards,
Johan

**Comment by kubk** [comment, marked as answer]
Hi, the up-to-date documentation is here: https://mobx.js.org/README.html

Thanks for the report, the old links should be removed. Related PR: https://github.com/mobxjs/mobx/pull/3603

**Reply by jeggers-88** [reply]
Thanks!
